eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import argparse
+
+import pdo.common.logger as plogger
+
+import pdo.eservice.pdo_helper as pdo_enclave_helper
+import pdo.eservice.pdo_enclave as pdo_enclave
+import logging
+logger = logging.getLogger(__name__)
+
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+def LocalMain(save_path) :
+    try :
+        logger.debug('initialize the enclave')
+        pdo_enclave_helper.get_enclave_service_info()
+
+        logger.info('save MR_ENCLAVE and MR_BASENAME to %s', save_path)
+        with open(save_path, "w") as file :
```

Comment:
Please check that "w" (as opposed to "w+") is sufficient.

Reply:
"w" should be fine, since we are not doing a read.